src/inference/hmc.jl

```diff
@@ -122,7 +122,7 @@ function sample_init!(
     if spl.alg isa AdaptiveHamiltonian
         # If there's no chain passed in, verify the n_adapts.
         if resume_from === nothing
-            if spl.alg.n_adapts == 0
+            if spl.alg.n_adapts == -1
```

Comment:
Maybe add a comment here, explaining the meaning of `-1`?

Reply:
I added one, thanks!